tests/cursor/cursor-tailable-001.phpt

```diff
@@ -0,0 +1,86 @@
+--TEST--
+MongoDB\Driver\Cursor tailable iteration
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php NEEDS('STANDALONE'); CLEANUP(STANDALONE); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+function insert(MongoDB\Driver\Manager $manager, $from, $to = null)
+{
+    if ($to === null) {
+        $to = $from;
+    }
+
+    $bulkWrite = new MongoDB\Driver\BulkWrite;
+
+    for ($i = $from; $i <= $to; $i++) {
+        $bulkWrite->insert(['_id' => $i]);
+    }
+
+    $writeResult = $manager->executeBulkWrite(NS, $bulkWrite);
+
+    printf("Inserted %d document(s): %s\n", $writeResult->getInsertedCount(), implode(range($from, $to), ', '));
+}
+
+$manager = new MongoDB\Driver\Manager(STANDALONE);
+
+$manager->executeCommand(DATABASE_NAME, new MongoDB\Driver\Command([
+    'create' => COLLECTION_NAME,
+    'capped' => true,
+    'size' => 1048576,
+]));
+
+insert($manager, 1, 3);
+
+$cursor = $manager->executeQuery(NS, new MongoDB\Driver\Query([], ['tailable' => true]));
+$it = new IteratorIterator($cursor);
+
+$numAwaitAttempts = 0;
+$maxAwaitAttempts = 7;
+
+for ($it->rewind(); $numAwaitAttempts < $maxAwaitAttempts; $it->next()) {
```

Comment:
@alcaeus: This logic may be relevant for [mongo-php-adapter](https://github.com/alcaeus/mongo-php-adapter).